crates/nu-std/lib/help.nu

```diff
@@ -138,16 +139,16 @@ def show-module [module: record] {
     }
 
     if not ($module.aliases? | is-empty) {
-        print-help-header -n "Exported aliases:"
-        print $module.aliases
+        print-help-header "Exported aliases"
+        print $"    ($module.aliases | str join ', ')"
```

Comment:
> Also fixed up the display of exported aliases to match help. You're welcome.
>
out of the scope of this PR but ehe, it looks better :smirk: :sparkles: 

thanks for this simple fix :blush: